.golangci.yml

```diff
@@ -0,0 +1,70 @@
+linters:
```

Comment:
right now we have linter running outside of CI, but with different config, file a separate issue: https://github.com/doublecloud/transfer/issues/177